components/table/TaskTable.tsx

```diff
@@ -123,6 +129,39 @@ function TaskTable({ project, tasks }: TaskTableProps): JSX.Element {
                             />
                         </ModalContent>
                     </Modal>
+                    <Modal
+                        isOpen={Boolean(taskToEdit)}
+<<<<<<< HEAD
+=======
+                        // eslint-disable-next-line no-undefined
+>>>>>>> dfb3db72dc25901dfb23804243b00d8bb38cef35
+                        onClose={() => setTaskToEdit(undefined)}
+                    >
+                        <ModalOverlay />
+                        <ModalContent px="0.5rem">
+                            <ModalHeader>Edit task</ModalHeader>
+                            <ModalCloseButton />
+                            {taskToEdit && (
+                                <EditTaskForm
+                                    task={taskToEdit}
+                                    project={project}
+                                    projectId={project.id}
+                                    afterSubmit={(taskUpdate) =>
+                                        taskUpdate.isSuccess &&
+<<<<<<< HEAD
```

Comment:
Tänne on jäänyt näitä gitin tuottamia merge-conflict -markereita.

Nä pitäis selvittää, jotta tätä voisi alkaa katselmoida kunnolla.